gno.land/pkg/sdk/vm/builtins.go

```diff
@@ -16,7 +16,7 @@ func (vm *VMKeeper) initBuiltinPackagesAndTypes(store gno.Store) {
 	// NOTE: native functions/methods added here must be quick operations,
```

Comment:
There's no real need to have this function at all... getPackage below might as well be a method on the store.